src/arch/LowLevelWindow/LowLevelWindow_SDL.cpp

```diff
@@ -0,0 +1,165 @@
+#include "global.h"
+#include "LowLevelWindow_SDL.h"
+#include "RageLog.h"
+#include "RageException.h"
+#include "archutils/Unix/X11Helper.h"
```

Comment:
If this is to be used with Windows, I don't think this header is the best idea. What specifically is used within this header?